test/other/benchmarkMemPoolTypes.cpp

```diff
@@ -0,0 +1,87 @@
+#include "System/MemPoolTypes.h"
+#include "System/Log/ILog.h"
+
+#include <benchmark/benchmark.h>
+
+#include <array>
+#include <cstddef>
+#include <vector>
+
+template <size_t T>
+struct ArrayData {
+	char Data[T-8];
```

Comment:
Is the 8 actually `sizeof ArrayData::idx`? Consider writing that out (perhaps as `sizeof(size_t)` if `idx` is not yet in scope or something).

Reply:
fixed